.golangci.yml

```diff
@@ -1,3 +1,4 @@
+# SPDX-FileCopyrightText: 2024 Intel Corporation
```

Comment:
Sometimes you put latest copyright at bottom and this time you are keeping it at top. Could you findout what is best practise?

Reply:
I was trying to follow this format/convention: https://github.com/omec-project/gnbsim/blob/main/gnbsim.go#L1